.travis.yml

```diff
@@ -17,5 +17,5 @@ script:
 env:
   global:
     - GO111MODULE=on
-    - KIND_VERSION=v0.3.0
-    - KUBERNETES_VERSION=1.13.6
+    - KIND_VERSION=v0.5.1
+    - KUBERNETES_VERSION=1.13.10
```

Comment:
```suggestion
    - KUBERNETES_VERSION=1.13.11
```

Reply:
Kind does not support `1.13.11`, the latest `v1.13` is `v1.13.10`